### ISSUE TITLE
Defining a props as an Object.

### ISSUE THREAD

**Issue by sagarchoudhary96** [measuredco/puck#79]
i have a Component with props as 
```
{
 name: string;
price: {"total": number; "string": number}
}
```
@chrisvxd 
how can i define the config for the price prop for puck editor. i see we added support for "custom" type but not sure if this can be done with it .

**Comment by chrisvxd** [comment]
Hi @sagarchoudhary96, we don't support objects yet. See #60 and #62.

TLDR we decided to wait on supporting `object` types until we've implemented #64 because we're not sure the `array` field UI makes sense.

Suggest you could flatten your object to `priceTotal` and `priceString` for the mean time.

**Comment by chrisvxd** [comment]
@sagarchoudhary96 I ended up doing this in #256. Should be available in `0.13.0-canary.b95050d`. Please let me know if that works for you.
